scripts/test_path_preservation.ps1

```diff
@@ -0,0 +1,148 @@
+# This test installs emsdk and activates the latest toolchain using `--system` or `--permanent` flags, and checks if parts of PATH are lost or overwritten. Set $env:SYSTEM_FLAG and $env:PERMANENT_FLAG to test each. If no flag is provided the process/shell values are tested. See the CI file for an example.
```

Comment:
Can you wrap these this line at 80 columns.

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/642/commits/9b6acc9ee7cd917b196dd826c845010b34120fff